validphys2/src/validphys/pdfgrids.py

```diff
@@ -374,6 +374,42 @@ def distance_grids(pdfs, xplotting_grids, normalize_to: (int, str, type(None)) =
 
     return newgrids
 
+@check_pdf_normalize_to
+def pull_grids(pdfs, xplotting_grids, normalize_to: (int, str, type(None)) = None):
+    """Return an object containing the value of the pull between the two PDFs at the specified values
+    of x and flavour.
+    The parameter ``normalize_to`` identifies the reference PDF set with respect to the
+    pull is computed.
+    This method returns pull grids where the relative pull between both PDF
+    set is computed. At least one grid will be identical to zero.
```

Comment:
```suggestion
    sets, defined as the distance in terms of the standard deviations of the reference PDF, is computed. At least one grid will be identical to zero.
```

Or whatever explanation is more suitable, but it would be good to provide the definition of the pull that is computed.